### PR TITLE
Feature: new drone steps for telegram notification and docs

### PR DESCRIPTION
**Pull request by Kohelbekker**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -58,6 +58,63 @@ steps:
...
 
+- name: docs-generate
+  image: quay.io/openware/sdk-tools:test-fetch1
+  environment:
+    BOT_USERNAME: kite-bot
+    BOT_NAME: Kite Bot
+    BOT_EMAIL: kite-bot@heliostech.fr
+    REPO_NAME: ${DRONE_REPO}
+    BRANCH_NAME: ${DRONE_BRANCH}
+    SDK_BRANCH: test
```
Comment (chumaknadya) [inline .drone.yml]: SDK_BRANCH=${DRONE_BRANCH}

File: .drone.yml [hunk 2]
```diff
@@ -58,6 +58,63 @@ steps:
...
+    GITHUB_API_KEY:
+      from_secret: kite_bot_key
+  commands:
+    - cd /sdk
+    - bundle exec rake ci:pull
+    - bundle exec rake ci:changelog['/drone/src/docs']
```
Comment (chumaknadya) [inline .drone.yml]: - bundle exec rake ci:changelog['/drone/src/']

File: .drone.yml [hunk 3]
```diff
@@ -58,6 +58,63 @@ steps:
...
+    GITHUB_API_KEY: 
+      from_secret: kite_bot_key
+    RAILS_MASTER_KEY:
+      from_secret: rails_master_key
+    SLACK_TOKEN:
+      from_secret: slack_token
+    SLACK_CHANNEL:
+      from_secret: slack_channel
+  commands:
+      - BUNDLE_GEMFILE=/sdk/Gemfile bundle exec rake --rakefile=/sdk/Rakefile sdk:notify:telegram
```
Comment (chumaknadya) [inline .drone.yml]: - BUNDLE_GEMFILE=/sdk/Gemfile bundle exec rake --rakefile=/sdk/Rakefile sdk:notify:telegram sdk:notify:slack

File: .drone.yml [hunk 4]
```diff
@@ -58,6 +58,63 @@ steps:
...
+      - BUNDLE_GEMFILE=/sdk/Gemfile bundle exec rake --rakefile=/sdk/Rakefile sdk:notify:slack
```
Comment (chumaknadya) [inline .drone.yml]: delete this line

File: .drone.yml [hunk 5]
```diff
@@ -40,9 +40,9 @@ steps:
     BOT_USERNAME: kite-bot
     BOT_NAME: Kite Bot
     BOT_EMAIL: kite-bot@heliostech.fr
-    BRANCH_NAME: ${DRONE_BRANCH}
+    BRANCH_NAME: master
```
Comment (chumaknadya) [inline .drone.yml]: BRANCH_NAME: ${DRONE_BRANCH}

File: .drone.yml [hunk 6]
```diff
@@ -58,6 +58,63 @@ steps:
...
+    - bundle exec rake ci:pull
+    - bundle exec rake ci:changelog['/drone/src/docs']
+    - bundle exec rake ci:copy_docs['/drone/src/docs']
+    - bundle exec rake ci:push
+  when:
+    event:
+      - push
+    branch:
+      - master
+      - 2-0-stable
```
Comment (chumaknadya) [inline .drone.yml]: Please, check stable branches on barong. We don't have 2-0-stable branch on barong

**Review by chumaknadya — CHANGES_REQUESTED** [review]

**Review by mod — APPROVED** [review]

**Pull request merged**